### PR TITLE
Feature/foo6and7

### PR DESCRIPTION
**Pull request by max-gun**
this is my implementation for foo 6 and foo 7

### CODE REVIEW

File: src/main/java/com/enterprise/testings/util/Exchanger.java [hunk 1]
```diff
@@ -25,4 +25,12 @@ public static void foo5() {
         System.out.println("kung foo5");
         System.out.println("jackie chan");
     }
+
+    public static void foo6() {
+        System.out.println("kung foo fighters");
```
Comment (db0206) [inline src/main/java/com/enterprise/testings/util/Exchanger.java:30]: Add Jet Li

File: src/main/java/com/enterprise/testings/util/Exchanger.java [hunk 2]
```diff
@@ -25,4 +25,12 @@ public static void foo5() {
         System.out.println("kung foo5");
         System.out.println("jackie chan");
     }
+
```
Comment (krasnie22) [inline src/main/java/com/enterprise/testings/util/Exchanger.java:28]: please print logs 2

**Review by db0206 — COMMENTED** [review]
Add Jet Li

File: src/main/java/com/enterprise/testings/util/Exchanger.java (hunk 1 above)
Comment (max-gun) [reply src/main/java/com/enterprise/testings/util/Exchanger.java:30]: Done!

**Review by db0206 — APPROVED** [review]

**Review by krasnie22 — APPROVED** [review]

**Pull request merged**